tests/components/image_processing/test_microsoft_face_identify.py

```diff
@@ -164,3 +164,22 @@ def mock_face_event(event):
         assert face_events[0].data['confidence'] == float(92)
         assert face_events[0].data['entity_id'] == \
             'image_processing.test_local'
+
+        # Test that later, if a request is made that results in no face
+        # being detected, that this is reflected in the state object
+        aioclient_mock.clear_requests()
+        aioclient_mock.post(
+            self.endpoint_url.format("detect"),
+            text=load_fixture('microsoft_face_detect_no_face_detected.json')
```

Comment:
please use `text="[]"` and remove this json file without a input

Reply:
👍